main.go

```diff
@@ -144,6 +144,8 @@ func main() {
 		mappingConfig   = kingpin.Flag("statsd.mapping-config", "Metric mapping configuration file name.").String()
 		readBuffer      = kingpin.Flag("statsd.read-buffer", "Size (in bytes) of the operating system's transmit read buffer associated with the UDP connection. Please make sure the kernel parameters net.core.rmem_max is set to a value greater than the value specified.").Int()
 		dumpFSMPath     = kingpin.Flag("debug.dump-fsm", "The path to dump internal FSM generated for glob matching as Dot file.").Default("").String()
+
+		cacheSize = kingpin.Flag("statsd.cache-size", "Maximum size of your metric mapping cache. Relies on least recently used replacement policy if max size is reached.").Default("1000").Int()
```

Comment:
I would keep the LRU out of the usage and document it in the README maybe?

Reply:
the unit is not immediately clear from this – "size" suggests bytes but I think it's the number of cached mappings?

```suggestion
		cacheSize = kingpin.Flag("statsd.cache-size", "Maximum number of cached mappings.").Default("1000").Int()
```

Reply:
You've got it!